lib/restful_model_collection.rb

```diff
@@ -15,16 +15,17 @@ def initialize(model_class, api, filters = {})
     def each
       return enum_for(:each) unless block_given?
 
-      offset = 0
-      chunk_size = 100
+      @filters[:offset] = 0
```

Comment:
Ah, I think you're right.

I added an `unless @filters.key?(:offset)` for both offset and limit - that way we have defaults, but if something has already been set `each` will respect those parameters.